hypervisor/include/common/schedule.h

```diff
@@ -28,6 +28,14 @@ enum sched_notify_mode {
 	SCHED_NOTIFY_IPI
 };
 
+/* Tools can configure a VM to use PRIO_LOW or PRIO_HIGH */
+enum thread_priority {
+	PRIO_IDLE = 0,
+	PRIO_LOW,
+	PRIO_HIGH,
+	PRIO_MAX
```

Comment:
Can you document how PRIO_IDLE and PRIO_MAX are used?  Is the thread priority statically defined and can't be changed? I don't see a function to change a thread scheduling priority?

Reply:
The PRIO_IDLE and PRIO_MAX are not exposed to users for configuration. They are only for ACRN coding. A VM chooses to use "SCHED_PRIO" can be confgured to use either PRIO_LOW or PRIO_HIGH.